doc/input-output-reference/src/overview/group-solar-collectors.tex

```diff
@@ -568,6 +568,117 @@ \subsubsection{Inputs}\label{inputs-5-029}
   0.84 ;                                                     !- Front Surface Emittance
 \end{lstlisting}
 
+\subsection{SolarCollectorPerformance:PhotovoltaicThermal:BIPVT}\label{solarcollectorperformancephotovoltaicthermalBIPVT}
+
+This object is used to provide performance details for the Building-Integrated PVT (BIPVT) model. There are no specific output variables for this object, reporting is done by the parent PVT object.
```

Comment:
"parent PVT object". It would be helpful to spell out the full object type here with hyperref.